stdlib/tests/crypto/falcon.rs

```diff
@@ -191,6 +191,8 @@ fn falcon_prove_verify() {
     let advice_inputs = AdviceInputs::default().with_map(advice_map);
     let advice_provider = MemAdviceProvider::from(advice_inputs);
     let mut host = DefaultHost::new(advice_provider);
+    host.load_mast_forest(StdLibrary::default().mast_forest().clone())
+        .expect("failed to load mast forest");
```

Comment:
How was it working previously without this?

Reply:
The test was ignored, so we should probably now remove `#[ignore]`?

Reply:
`#[ignore]` removed